conans/client/generators/cmake_find_package.py

```diff
@@ -19,14 +19,15 @@ class CMakeFindPackageGenerator(Generator):
 
         include(FindPackageHandleStandardArgs)
 
-        conan_message(STATUS "Conan: Using autogenerated Find{name}.cmake")
+        conan_message(STATUS "Conan: Using autogenerated Find{filename}.cmake")
         # Global approach
-        set({name}_FOUND 1)
+        set({filename}_FOUND 1)
```

Comment:
This will be braking for packages that don't use `cpp_info.filename`

Reply:
`filename` defaults to `cpp_info.names['generator']` which defaults to cpp_info.filename, so this should be safe.